src/__internal__/fieldset/fieldset.style.js

```diff
@@ -13,9 +13,18 @@ const StyledFieldset = styled.fieldset`
     ml
   }) => ml && `margin-left: ${ml};`}
 
+${({
+    mb,
+    theme
+  }) => mb && css`margin-bottom: ${mb * theme.spacing}px;`}
+
```

Comment:
I suggest to rewrite it to
```js
${({ml, mb, theme}) => css`
  border: none;
  margin: 0;
  padding: 0;
  min-width: 0;
  min-inline-size: 0;

  ${ml && `margin-left: ${ml}`}
  ${mb && `margin-bottom: ${mb * theme.spacing}px;`}
  ${({ styleOverride }) => styleOverride}
`}
```
I think it will be easier to read the code

Reply:
I agree I'll change it